### DISCUSSION TITLE
Sending key presses to background workspaces

### DISCUSSION THREAD

**Discussion by febs** [i3/i3#5696] [Q&A]
Is it possible by any mean to send key presses to a workspace that is not currently active and shown?

**Comment by okraits** [comment]
What is the actual usecase?

**Reply by febs** [reply]
a certain application requires regular input not to become inactive and quit. I would like to keep that window active, without the need to actually switch focus.

**Reply by okraits** [reply]
Maybe you can use `xdotool` for that.
